walk-in-interview/src/main/webapp/new-job.html

```diff
@@ -60,7 +60,7 @@
 
             <div class="job-field">
                 Post Expiry:
-                <input type="date" id="new-post-expiry" name="post-expiry" 
+                <input type="date" id="new-job-expiry" name="job-expiry" 
                     min="2020-07-01" max="2021-06-30" required>
                 <!-- TODO(issue/11) -->
```

Comment:
nit, please indicate some details about the issue not just the issue Id. something like `TODO(issue/11) do this thing -->`